integration_tests/test_complex.py

```diff
@@ -11,7 +11,7 @@ def test_real_imag():
     b = x.imag
     assert abs(a - 2) < eps
     # TODO: below test should work
-    # assert abs(b - 3) < eps
+    assert abs(b - 3) < eps
```

Comment:
Some more tests may be? For better coverage.

Reply:
Do you have any tips as to how I can write some tests to check whether an implicit cast node has a compile time value?

Reply:
I think value calculation will happen while visiting `ImplicitCast` node. So, adding tests which will require implicit cast should do the job. Like, `c = a + b` where `a`, `b` and `c` all are of different types.